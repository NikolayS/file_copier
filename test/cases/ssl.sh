#!/bin/bash

# to run this scenario, COPIERHOST should be defined (`export COPIERHOST=http://your.copier.host.name`)

http GET "http://$COPIERHOST/?wh=1&md5=1&src=https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png" -h | grep -v "Date:"

