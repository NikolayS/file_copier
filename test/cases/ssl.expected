HTTP/1.1 200 OK
Server: nginx
Transfer-Encoding: chunked
X-File-Copier-Md5: 80fa4bcab0351fdccb69c66fb55dcd00
X-File-Copier-Img-Width: 544
X-File-Copier-Img-Height: 184
X-File-Copier-Size: 13504
X-Location: /data/26/20/84/25/262084257c2103702ef8a25705e3f8dbc1fa3823103ad7b954d54bdb77e6d89d.png
Access-Control-Expose-Headers: X-File-Copier-Size, X-Location, X-File-Copier-Img-Height, X-File-Copier-Img-Width, X-File-Copier-Md5

